style_rwd.css

```diff
@@ -2,21 +2,233 @@
     img.main_photo {
         display: none;
     }
-    .catch_phrase{
-        max-width: 767px;
+    h2 {
+        margin-block-start: 0;
+        margin-block-end: 0;
+    }
+    .nav_margin {
+        margin: 0 ;
+    }
+    .margin_header{
+        margin: 0;
+    }
+    .margin_menu {
+        margin: 0;
+    }
+    .header_wrapper>div {
+        padding: 5px 0 5px 10px;
+    }
+    .header_wrapper{
+        max-width:767px;
+        max-height:50px;
+    }
+    .section_wrapper{
+        max-width:767px;
+        margin:0 auto;
+        padding: 0;
+    }
+    .font_size{
+        font-size:10px;
+    }
+    .section_hero {
+        padding-bottom: 42px;
     }
     .column{
+        max-width:767px;
         align-items: center;
         justify-content: center;
     }
-    .invite_phrase{
+    .catch_phrase .invite_phrase{
         text-align: center;
-        font-size: 5px;
-        font-weight: 30;
-        max-width: 767px;
+        font-size: 40px;
+        margin-bottom: 0;
+        max-width:767px;
```

Comment:
why? 🤔

Reply:
why what? :) needed to change the font size here :)

Reply:
why do you set max-width 767px to a style which applies only below 768px 🤔

Reply:
ou thought it is necessary to give this information in here aswell... :) to make sure that it won't overflow

Reply:
it won't overflow if you don't set static width (greater than current viewport)

Reply:
got it :)